source/mapping-rules/field-customizations/field-customizations.txt

```diff
@@ -0,0 +1,71 @@
+.. _rm-field-customizations:
+
+====================
+Field Customizations
```

Comment:
Rename Field Customizations > Fields, move the calculated fields into the "FIelds Drawer".